src/app/_components/BreadCrumbsWrapper.tsx

```diff
@@ -8,10 +8,16 @@ import { PATHS } from '@/constants/paths'
 
 export function BreadCrumbsWrapper() {
   const pathname = usePathname()
+  const allPaths = Object.values(PATHS).map((route) => route.path) as string[]
```

Comment:
This wouldn't catch dynamic paths.

Reply:
good catch!

Reply:
For blog posts and case studies - could we add a condition in blog and case study pages to render the 404 if blog/case study not found? At the moment we have:

```
 if (!featuredPost) {
    throw new Error('Featured post not found')
  } 
```